frontend/src/hooks/useTrafficData.ts

```diff
@@ -172,7 +172,9 @@ const toMonthlyChartData = (
     );
 
     const labels = Array.from({ length: numMonths }).map((_, index) =>
-        formatMonth(addMonths(from, index)),
+        index === numMonths - 1
+            ? 'Current month'
+            : formatMonth(addMonths(from, index)),
```

Comment:
we want to mark the current month (that hasn't been completed) as such, so that there's no confusion.